unit-tests/func/playback/test-non-realtime.py

```diff
@@ -0,0 +1,31 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+import os
+import pyrealsense2 as rs2
+from rspy import test
+
+# This test checks that stop of pipeline with playback file
+# and non realtime mode is not stuck due to deadlock of
+# pipeline stop thread and syncer blocking enqueue thread (DSO-15157)
+#############################################################################################
+test.start("Playback with non realtime doesn't stuck at stop")
+current_dir = os.path.dirname( os.path.abspath( __file__ ))
+filename = current_dir + os.sep + 'recording.bag'
```

Comment:
Where is the file put??

Reply:
Shouldn't it be called `recording_deadlock.bag`? I'm confused...

Reply:
fixed